src/components/IFXContactablesCombobox.vue

```diff
@@ -87,6 +87,9 @@ export default {
     getItemValue(item) {
       return item
     },
+    removeFromSelected(item) {
+      this.selected.splice(this.selected.findIndex((i) => i.id === item.id), 1)
```

Comment:
Can this `findIndex` ever fail to find the item? If it returns -1, it will remove the last item from the list.

Reply:
Good question.  I can't think of a circumstance were it would fail; it's run when the "close" on a chip is clicked.  it would have to be there for it to be clicked.

Reply:
It shouldn't fail.  It's being called by the "x" on a chip.